conda/cli/main_info.py

```diff
@@ -133,9 +140,9 @@ def get_user_site():  # pragma: no cover
     return site_dirs
 
 
-IGNORE_FIELDS = {"files", "auth", "preferred_env", "priority"}
+IGNORE_FIELDS: set = {"files", "auth", "preferred_env", "priority"}
```

Comment:
```suggestion
IGNORE_FIELDS: set[str] = {"files", "auth", "preferred_env", "priority"}
```

Reply:
Of course, how could I forget!  Updated.